.circleci/config.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
   deploy_development:
     docker:
-      - image: circleci/node:8.11.2@sha256:495a13db597b5306f05ae728489fedb9faa4defbac24ca0e00e8cc1eca95396d
+      - image: circleci/node:10.18.1@sha256:8230f70c03b6ecf521a34125fb090ec4b47068205454e5820859c00260c9bd23
```

Comment:
got this information from [this](https://hub.docker.com/layers/circleci/node/10.18.1/images/sha256-8230f70c03b6ecf521a34125fb090ec4b47068205454e5820859c00260c9bd23)

Reply:
That page is leading nowhere. I looked up the new hashes from https://circleci.com/docs/2.0/circleci-images/#nodejs (actually at https://hub.docker.com/r/circleci/node/tags?page=1&name=10.18.1 ) and found 9b128e988f1df2c02b77f0f85df04e59ffb7de8ab07f21abedad9a8a20a29c3d .

I guess that it keeps getting updated, so I don't mind the hash, as long as the build passes.

Reply:
oh hmm.. i see. Yes you must be right - it keeps getting updated 🤷‍♀ 

ok cool. @caitmuenster  - is it okay to merge and deploy this to stage? or is there any reason to wait?

Reply:
Thanks @rebmullin! Yes, please deploy to stage!